### DISCUSSION TITLE
TraceQL query on attribute list

### DISCUSSION THREAD

**Discussion by pradaelli** [grafana/grafana#70309] [Observability]
I have an attribute of a span that is a list of strings (see image). 
![Screenshot 2023-06-19 at 10 43 14](https://github.com/grafana/grafana/assets/10598850/460efed6-513a-4136-ba5f-d114816e2720)

I would like to be able to run queries like `{ "att1" in .list_att }` or similar on these lists. Any suggestion on how to perform this? If it's not possible atm it would be a very nice feature!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.

**Reply by devurandom** [reply]
@pradaelli Did you open a question on the forum? I could not [find anything related](https://community.grafana.com/search?q=traceql+list) sadly.

I created https://github.com/grafana/grafana/issues/85682 as a feature request for this.
